lte/gateway/docker/example/subscriberdb.yml

```diff
@@ -0,0 +1,58 @@
+---
+#
```

Comment:
Are all example files needed?

Reply:
no, I removed them in a previous commit.